pkg/storaged/format-dialog.jsx

```diff
@@ -53,7 +56,7 @@ function extract_option(split, opt) {
     }
 }
 
-function mounting_dialog_fields(is_custom, mount_dir, mount_options, visible) {
+function mounting_dialogx_fields(is_custom, mount_dir, mount_options, visible) {
```

Comment:
OOI, why the renaming?

Reply:
The react dialog stuff is generally called `...x`, like `dialogx.jsx`.  I plan to remove the `x` suffix once the old-style dialog code is fully gone.